tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,6 +83,13 @@ func (tt *tester) runLoop() {
 			plog.Printf("[round#%d case#%d] recovered failure", i, j)
 
 			if !tt.consistencyCheck {
+				plog.Printf("[round#%d case#%d] restarting the stressers...", i, j)
```

Comment:
below, it cancels (stops) stressers. I will rework on this. (remove unnecessary debugging lines)